tox.ini

```diff
@@ -0,0 +1,93 @@
+[tox]
+#isolated_build = True
+envlist = black-check, isort-check, vulture, flake8, pylint, py3.9, py3.8, py3.7
+
+[tox:.package]
+# note tox will use the same python version as under what tox is installed to package
+# so unless this is python 3 you can require a given python version for the packaging
+# environment via the basepython key
+basepython = python3
+
+[testenv]
+setenv =
+    PYTHONPATH = {toxinidir}
+deps =
+    pytest
+    pytest-cov
+    pytest-randomly
+    ; Other test dependencies
+    ; ...
+    ; mistletoe>=0.7.2,<0.8.0
+    ; mesa>=0.8.9,<0.9.0
+
+; to include the "dadapy" folder: --doctest-modules dadapy tests/
+commands =
+    pytest --basetemp={envtmpdir} --doctest-modules tests/ \
+        --cov=dadapy \
+        --cov-report=xml \
+        --cov-report=html \
+        --cov-report=term
+
+[testenv:py3.7]
+basepython = python3.7
+
+[testenv:py3.8]
+basepython = python3.8
+
+[testenv:py3.9]
+basepython = python3.9
+
+[testenv:flake8]
+skip_install = True
+deps =
+    flake8
+    flake8-bugbear
+    flake8-docstrings
+    flake8-eradicate
+    flake8-isort
+    pydocstyle
+commands =
+    flake8 dadapy tests
+
+[testenv:pylint]
+skipdist = True
+deps = pylint
+commands = pylint dadapy tests
+
+[testenv:black]
+skip_install = True
+deps = black==20.8b1
+commands = black dadapy tests
+
+[testenv:black-check]
+skip_install = True
+deps = black==20.8b1
```

Comment:
Latest `black` version is `21.12b0`:

https://pypi.org/project/black/

Need to check whether it works well with isort